_docs/setup/cluster-extension.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Istio Kubernetes Cluster Extension
+overview: Instructions to add external machines to your Istio managed Kubernetes cluster.
+
+order: 25
+
+layout: docs
+type: markdown
+---
+
+This task shows how to install Istio in a non-kubernetes machine in the same network
+(VPC, VPN) as the Kubernetes cluster.
+
```

Comment:
this is not a task. Please add an overview, some background, a diagram preferably, along with recommended install path. Otherwise, this section feels like odd man out, without background context

Reply:
there are other nits, but since this is a wip document, let me know once its done

Reply:
Updated.